### PR TITLE
Added a help-block string to the 'File Versioning' tab

### PR DESCRIPTION
**Pull request by ralienpp**
### Purpose

Provide a bit more information to users when they create new folders, to clarify that "File versioning" can be related to file deletion, not just overwriting. 

A broader context is available on the forum: https://forum.syncthing.net/t/missing-syncthing-directory-missing-data-on-devices/13684/9?u=ralienpp

### Testing

None, I looked at the code of the page and added this string, in a way analogous to other tabs.

### CODE REVIEW

File: gui/default/syncthing/folder/editFolderModalView.html [hunk 1]
```diff
@@ -65,6 +65,9 @@
         <div id="folder-versioning" class="tab-pane">
           <div class="form-group">
             <label translate>File Versioning</label>&emsp;<a href="https://docs.syncthing.net/users/versioning.html" target="_blank"><span class="fas fa-question-circle"></span>&nbsp;<span translate>Help</span></a>
+            <p class="help-block">
+              <span translate>What to do if files are replaced or deleted, either locally, or on remote devices.</span>&emsp;
```
Comment (imsodin) [inline gui/default/syncthing/folder/editFolderModalView.html]: This is wrong, as Syncthing can't version local deletes. Syncthing only reacts to changes the user makes locally, i.e. it sees the change after it happened. In case of a deletion, it just notices that there is nothing on disk where previously there was a file - so the data is gone already. Remote deletions on the other hand still exist locally, so instead of just deleting the files, Syncthing can use them for versioning.

Comment (AudriusButkevicius) [reply gui/default/syncthing/folder/editFolderModalView.html]: Also this sentence triggers my grammatical spider senses. Not sure what the goal of the sentence is.

Comment (Catfriend1) [reply gui/default/syncthing/folder/editFolderModalView.html]: Why not take a short form of the explanation from the docs?
> Syncthing supports archiving the old version of a file when it is deleted or replaced with a newer version from the cluster. Click to see available versioning strategies. File versioning is configured per folder, on a per-device basis.

Comment (ralienpp) [reply gui/default/syncthing/folder/editFolderModalView.html]: Thank you for the input and the corrections. The explanation from the docs is better, but I think it would make sense, especially for the Android client, to make it shorter.

The goal of the sentence is to make users aware of the relationship between versioning and deletion.

How about a slightly more concise wording:

- I removed "per folder", because we're already looking at a folder-specific dialog, thus it is of reason to expect that the settings are folder-specific
- "Click to see..." is removed to shorten the text, but I don't insist on adding it

> Syncthing supports archiving the old version of a file when it is deleted or replaced with a newer version from the cluster. File versioning is configured on a per-device basis.

image

Comment (imsodin) [reply gui/default/syncthing/folder/editFolderModalView.html]: I'd completely remove the "per-device" sentence - not even sure what it is trying to say. Then I'd replace "the cluster" with "remote devices", as it's constent with the rest of the UI and easier to understand. And starting with "Syncthing supports" in a Syncthing dialog feels redundant, I'd rather write something like "Choose a strategy to archive old versions of a files when they are deleted or replaced with a newer version from remote devices."

**Comment by AudriusButkevicius** [comment]
Hi,

Do you have any intentions of addressing the review comments, or should we close this?

Thanks.
A

**Comment by st-review** [comment]
:robot: *beep boop*

I'm going to close this pull request as it has been idle for more than 90 days.

This is not a rejection, merely a removal from the list of active pull requests that is periodically reviewed by humans. The pull request can be reopened when there is new activity, and merged once any remaining issues are resolved.
